### PR TITLE
fix Contact force Penalty

### PR DESCRIPTION
**Pull request by Hokyun-Lee**
I just corrected contact_force_penalty_thres code in below. (delete "1-" in code.)

However,
How about this formulation with keep using "1-" method?

contact_force_penalty_thres = **rew_weights["ContactForceThresholdPenalty"]** * (1-torch.exp(-0.007*(torch.norm(torch.clamp(lfoot_force[:,2].unsqueeze(-1) - 1.4*9.81*total_mass, min=0.0), dim=1) \
                                                            + torch.norm(torch.clamp(rfoot_force[:,2].unsqueeze(-1) - 1.4*9.81*total_mass, min=0.0), dim=1))))

contact_force_penalty = torch.where(thres.squeeze(-1), contact_force_penalty_thres[:], **zeros[:]**)

For detail, rew_weights["ContactForceThresholdPenalty"] will be negative value.
And then, rewards will be zero without contact force threshold violation.

in my case, rew_weights["ContactForceThresholdPenalty"] = -3.0 to -5.0 really works for me.

### CODE REVIEW

**Comment by Hokyun-Lee** [comment]
in case, y=e^(-0.007\*x)
and y = 1 at f_z < 1.4\*mass\*grav


incase, y=-(1-e^(-0.007\*x))
and y = 0 at f_z < 1.4\*mass\*grav

**Comment by kdh0429** [comment]
I think keeping "1-" and using negative weight does not make big difference. Gradients are the same in both cases, which determines the update

**Review by kdh0429 — APPROVED** [review]
Looks good to me
